Assignment2/patin9-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+	"@context": {
+		"assig": "http://assignment2.com/2b#",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"name": "assig:name",
+		"homeTown": "assig:homeTown",
+		"age": "assig:age",
+		"almaMater": "assig:almaMater",
+		"parentsName": "assig:parentsName",
+        "subjects": "assig:subjects"
+	},
+	"name": "Eva Tagarro",
```

Comment:
Add the. "@id" and "@type"

Reply:
Be careful when reusing prefixes. You don't have any assignment prefix on your context definitions. The correct definition is "@type": "assign:student"